system/utils.py

```diff
@@ -391,7 +391,10 @@ def wrapped(operations, queue_size, add_size, get_size, event_loop):
 # TODO make that async
 def check_no_failures(hosts):
     for host in hosts:
-        result = host.run('journalctl -u indy-node.service -b -p info')
+        try:
+            result = host.run('journalctl -u indy-node.service -b -p info')
+        except AssertionError:
+            result = ''
```

Comment:
I think it would be better to indicate somehow that we failed to retrieve journalctl due to some unexpected assertion

Reply:
Done.